Add XMPP Summit blogpost 29 + event (#1677)
xsf/xmpp.org: content/blog/summit/2026-05-21_xmpp-summit-29.md

---
title: XMPP Summit 29
date: 2026-05-21
author: XMPP Standards Foundation
categories: ["XMPP Summit", "Events"]
---

> The XMPP Standards Foundation (XSF) is excited to announce the 29th XMPP Summit as first summit taking place online!

Reviewer: "as first summit taking place online" is missing an article. It is also awkward phrasing. 
```suggestion
The XMPP Standards Foundation (XSF) is excited to announce the 29th XMPP Summit, the first XMPP Summit to take place fully online!
```

The XMPP Summit will be held from Friday 4th September to Saturday 5th September 2026, both days between 13:00 - 16:00 UTC.

> The XSF invites everyone interested in development of the XMPP protocol to attend, and discuss all things XMPP remotely!

Reviewer: This can be made slightly more inclusive maybe:

```suggestion
The XSF invites everyone interested in development of XMPP technologies to attend, and discuss all things XMPP remotely!
```

## The _online_ XMPP Summit

The XMPP Summit is where the XMPP community comes together to discuss protocol development (such as [XMPP extensions (XEPs)](https://xmpp.org/extensions/)), implementation experience, and the future of the ecosystem.

> As this is the first online XMPP Summit, its intentionally kept lightweight.

Reviewer: ```suggestion
As this is the first online XMPP Summit, it's intentionally kept lightweight.
```

The goal is not to replicate the in-person XMPP Summit experience, but to provide value that complements our existing asynchronous communication channels.

Favored topics will likely be follow-ups to topics discussed during Summit 28, advance current work on extensions (XEPs), discuss implementation pain points, and improvement of collaboration between projects.
In general, the intention is to make it easier for more XMPP community members to participate, too.

### Info, Date & Time

> Date:  Friday 4th - Saturday 5th, September 2026
> 
> Time: 13:00 - 16:00 UTC (both days)
> 
> Session link: [TBA](https://wiki.xmpp.org/web/Conferences/Summit_29#Platform)
> 
> Cost: Free (registration and topic proposals appreciated)  

### Who participates at the Summit?

Similar to an _unconference_ at the beginning all interested participants can suggest topics and others can indicate via votes whether or not they are interested in these topics.
Afterwards a rough order of topics is established that will be followed in moderation with the participants.

### How does the Summit work?

> If you have ever followed a thread on the [Standards Mailing List](https://xmpp.org/community/mailing-lists/) or taken part in a discussion on the [public XSF channel](https://xmpp.org/community/chat/) you should be familiar with this - just in person.

Reviewer: 'just in person' is not quite correct, because this Summit is online.

Author: d3a5bcb

The different topics are broken up by short breaks that are great for networking and getting to know other XMPP developers.

> Agreeing on a common strategy or even establishing a rough priority for certain features in our decentral and interoperable technology and protocol can be challenging.

Reviewer: ```suggestion
Agreeing on a common strategy or even establishing a rough priority for certain features in our decentralized and interoperable technology and protocol can be challenging.
```
